esp-hal/README.md

```diff
@@ -1,59 +1,55 @@
 # esp-hal
 
 [![Crates.io](https://img.shields.io/crates/v/esp-hal?labelColor=1C2C2E&color=C96329&logo=Rust&style=flat-square)](https://crates.io/crates/esp-hal)
+[![docs.rs](https://img.shields.io/docsrs/esp-hal?labelColor=1C2C2E&color=C96329&logo=rust&style=flat-square)](https://docs.esp-rs.org/esp-hal)
+![MSRV](https://img.shields.io/badge/MSRV-1.76-blue?labelColor=1C2C2E&style=flat-square)
 ![Crates.io](https://img.shields.io/crates/l/esp-hal?labelColor=1C2C2E&style=flat-square)
 [![Matrix](https://img.shields.io/matrix/esp-rs:matrix.org?label=join%20matrix&labelColor=1C2C2E&color=BEC5C9&logo=matrix&style=flat-square)](https://matrix.to/#/#esp-rs:matrix.org)
 
-The `esp-hal` package aims to provide a safe, idiomatic Hardware Abstraction Layer (HAL) for the entire family of ESP32 devices from Espressif.
+Bare-metal (`no_std`) hardware abstraction layer for Espressif devices.
 
-This package implements both blocking and, when able, asynchronous drivers for the various peripherals. At this time the blocking APIs are used by default, with all asynchronous functionality gated behind the `async` feature. See the package documentation for more information on its features.
+Implements a number of blocking and, where applicable, async traits from the various packages in the [embedded-hal] repository.
 
-Most traits defined by the [embedded-hal] family of packages are implemented as applicable.
+For help getting started with this HAL, please refer to [The Rust on ESP Book] and the [documentation].
```

Comment:
Should we also include a link here to the no_std training as getting started guide? https://docs.esp-rs.org/no_std-training/

Reply:
Possibly, I am not quite sure how much detail to go into. I think the book probably mentions the training? But I'm also not against adding it, again just trying to find a good balance here.